oem/firstvoices/ios/build_keyboards.sh

```diff
@@ -102,3 +109,27 @@ if [ $DO_BUILD = true ] || [ $DO_COPY = true ]; then
 
   popd
 fi
+
+if [ $DO_DOWNLOAD = true ]; then
+  echo "Downloading keyboards from downloads.keyman.com"
+
+  cp ../keyboards.csv "$TARGET/keyboards.csv"
+
+  SCRIPT_ROOT=`pwd`
+  URL_DOWNLOAD=https://downloads.keyman.com
+  URL_API_VERSION=${URL_DOWNLOAD}/api/keyboard/
+
+  {
+    # Skip header line
+    read
+
+    # Read CSV and build each referenced keyboard
+    while IFS=, read -r shortname id name region old_keyboard; do
+      echo "Downloading $id ($name)"
+      # Get latest version
+      URL_DOWNLOAD_FILE=`curl -s "$URL_API_VERSION/$id" | jq -r .js`
```

Comment:
I tried building locally and got the error
```
Downloading fv_inuvialuktun (Inuvialuktun)
./build_keyboards.sh: line 130: jq: command not found
```

Would this repo need a copy of keymanapp/keyboards/tools/jq-win64.exe ?

Reply:
If jq is standard on mac, maybe have the ios build check to only build on mac?

Reply:
Ah at present that's correct. There is disambiguation for jq in the keyboards repo. I'll do similar here for consistency.